server/helpers/checkCategory.js

```diff
@@ -0,0 +1,8 @@
+import models from '../models';
+
+const { Category } = models;
+
+const getRandomCategory = async () => (await Category
```

Comment:
Well-done @ekpangmichael. I think the code from line 5 to 6 will be more readable like this

```
const getRandomCategory = async () => (
    await Category.findAll({ attributes: ['id'] })
)[0].id;
```